uint8 servo_id
uint16 hw_min
uint16 hw_max
uint16 sw_min
uint16 sw_max
uint16 home
uint8 max_load_pct
uint8 max_temp
